src/avfoundation.cs

```diff
@@ -7552,10 +7642,29 @@ interface AVAssetExportSession {
 		[Field ("AVAssetExportPresetPassthrough")]
 		NSString PresetPassthrough { get; }
 
+		[NoWatch, NoTV, NoiOS, Mac (10,15)]
+		[Field ("AVAssetExportPresetAppleProRes4444LPCM")]
+		NSString PresetAppleProRes4444Lpcm { get; }
+
+		[TV (13,0), NoWatch, Mac (10,15), iOS (13,0)]
+		[Field ("AVAssetExportPresetHEVC1920x1080WithAlpha")]
+		NSString PresetHevc1920x1080WithAlpha { get; }
+
+		[TV (13,0), NoWatch, Mac (10,15), iOS (13,0)]
+		[Field ("AVAssetExportPresetHEVC3840x2160WithAlpha")]
+		NSString PresetHevc3840x2160WithAlpha { get; }
+
+		[TV (13,0), NoWatch, Mac (10,15), iOS (13,0)]
+		[Field ("AVAssetExportPresetHEVCHighestQualityWithAlpha")]
+		NSString PresetHevchHighestQualityWithAlpha { get; }
+
 		// 5.0 APIs
 		[Export ("asset", ArgumentSemantic.Retain)]
 		AVAsset Asset { get; }
 
+		[Deprecated (PlatformName.iOS, 13,0, message: "Use 'EstimateOutputFileLength' for more precise results.")]
+		[Deprecated (PlatformName.TvOS, 13,0, message: "Use 'EstimateOutputFileLength' for more precise results.")]
+		[Deprecated (PlatformName.MacOSX, 10,15, message: "Use 'EstimateOutputFileLength' for more precise results.")]
```

Comment:
same as earlier
add [NoWatch] here
there's no point in exposing the deprecated API in a new platform